packages/web-app-files/src/components/Search/List.vue

```diff
@@ -231,6 +256,27 @@ export default defineComponent({
       eventBus.publish('app.search.term.clear')
     })
 
+    // transifex hack
+    const lastModifiedTranslations = {
+      today: $gettext('today'),
+      yesterday: $gettext('yesterday'),
+      'this week': $gettext('this week'),
+      'last week': $gettext('last week'),
+      'last 7 days': $gettext('last 7 days'),
+      'this month': $gettext('this month'),
+      'last month': $gettext('last month'),
+      'last 30 days': $gettext('last 30 days'),
+      'this year': $gettext('this year'),
+      'last year': $gettext('last year')
+    }
+
+    const lastModifiedFilter = ref<VNodeRef>()
+    const availableLastModifiedValues = ref<LastModifiedKeyword[]>(
+      unref(modifiedDateCapability).keywords?.map(
+        (k: string) => ({ id: k, label: lastModifiedTranslations[k] } as LastModifiedKeyword)
```

Comment:
Shouldn't be necessary to cast `as LastModifiedKeyword` here, since you're correctly typed the ref and the capability, no?